### PR TITLE
Fix response schema for event stream

### LINKED ISSUE #86: Fix bug about creating the schema of EventStream response
First, I would like to thank you for the EventStream feature that came with the last commit.

I wanted to report a bug I've seen on that subject.

The schema is not added to the JSON file associated with the Event Stream.

```json
"response": {
           "200": {
             "description": "",
             "content": {
               "text/event-stream; charset=utf-8": {}
             }
           }
         },
```

so, Open Api Generator for typescript-fetch throws an error.

```
[main] ERROR o.o.codegen.DefaultCodegen - Undefined property/schema for `SchemaFor200ResponseBodyTextEventStream charsetutf8`. Default to type:string.
```

### PR DESCRIPTION
**Pull request by codercengiz**
Fix #86

### CODE REVIEW

File: rocket-okapi/src/gen.rs [hunk 1]
```diff
@@ -10,6 +10,7 @@ use std::collections::HashMap;
 
 /// A struct that visits all `rocket::Route`s, and aggregates information about them.
 #[derive(Debug, Clone)]
+#[allow(dead_code)]
```
Comment (svenstaro) [inline rocket-okapi/src/gen.rs:13]: Should we really allow this here? Why?

**Review by svenstaro — CHANGES_REQUESTED** [review]

File: rocket-okapi/src/gen.rs (hunk 1 above)
Comment (codercengiz) [reply rocket-okapi/src/gen.rs]: I understand your concern, so I used _ instead of ``#[allow(dead_code)]

**Comment by svenstaro** [comment]
@GREsau could you take a look at this please?

File: rocket-okapi/src/gen.rs [hunk 2]
```diff
@@ -11,7 +11,7 @@ use std::collections::HashMap;
 /// A struct that visits all `rocket::Route`s, and aggregates information about them.
 #[derive(Debug, Clone)]
 pub struct OpenApiGenerator {
-    settings: OpenApiSettings,
```
Comment (Nukesor) [inline rocket-okapi/src/gen.rs:14]: I assume this `_` has been a mistake? It doesn't seem relevant for this MR.

File: rocket-okapi/src/gen.rs [hunk 3]
```diff
@@ -11,7 +11,7 @@ use std::collections::HashMap;
...
+    _settings: OpenApiSettings,
```
Comment (Nukesor) [inline rocket-okapi/src/gen.rs:14]: I assume this _ has been a mistake? It doesn't seem relevant for this MR.

**Comment by svenstaro** [comment]
@GREsau would be great to get this fix in before the next release now that master is compatible to current Rocket. :)

File: rocket-okapi/src/gen.rs (hunk 3 above)
Comment (ralpha) [reply rocket-okapi/src/gen.rs:14]: Looks like this was just added to suppress an warning during compiling.

**Comment by ralpha** [comment]
Thanks for the PR, this is partially merged in 7acd774f5307a02396bfa4018d208bbb75e7a652.
Other changes are not needed anymore.
This should be fixed in the next release.

**Pull request closed without merging**